src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2280,6 +2280,190 @@ public override IOperation VisitThrow(IThrowOperation operation, int? captureIdF
             }
         }
 
+        public override IOperation VisitUsing(IUsingOperation operation, int? captureIdForResult)
+        {
+            Debug.Assert(operation == _currentStatement);
+
+            SemanticModel semanticModel = ((Operation)operation).SemanticModel;
+            ITypeSymbol iDisposable = semanticModel.Compilation.GetSpecialType(SpecialType.System_IDisposable);
+
+            if (operation.Resources.Kind == OperationKind.VariableDeclarationGroup)
```

Comment:
Can `Resources` be null for error cases?

Reply:
> Can Resources be null for error cases?

I don't think so, see TestOperationVisitor.VisitUsing.



---
In reply to: [178662063](https://github.com/dotnet/roslyn/pull/25828#discussion_r178662063) [](ancestors = 178662063)